### PR TITLE
Added the K8s Autodiscovery doc video

### PR DESCRIPTION
**Pull request by technovangelist**
adds the video to the docs site

### CODE REVIEW

File: content/videos/tipsandtricks-k8s-autodiscovery.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    
+summary: Orchestration platforms like Docker Swarm, Kubernetes, and Amazon ECS make running containerized services easier and more resilient by handling workload scheduling and request routing. But all of that makes monitoring more difficult. How can you reliably monitor a service which by design constantly changes the definition normal as it moves applications between hosts? The Datadog Agent can automatically track which services are running where, thanks to its Autodiscovery feature. This video will show you how easy it is to get started with Autodiscovery on Kubernetes with Datadog.
+---
+Orchestration platforms like Docker Swarm, Kubernetes, and Amazon ECS make running containerized services easier and more resilient by handling workload scheduling and request routing. But all of that makes monitoring more difficult. How can you reliably monitor a service which by design constantly changes the definition normal as it moves applications between hosts? The Datadog Agent can automatically track which services are running where, thanks to its Autodiscovery feature. This video will show you how easy it is to get started with Autodiscovery on Kubernetes with Datadog.
```
Comment (kmshultz) [inline content/videos/tipsandtricks-k8s-autodiscovery.md]: nit: the final "with Datadog" feels unnecessary.

File: content/videos/tipsandtricks-k8s-autodiscovery.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+* [Setting up the Guestbook Example](?wtime=77)
+* [Redis is Automatically Configured](?wtime=110)
+* [Configcheck](?wtime=128)
+* [Redis auto_conf](?wtime=149)
+* [Three Autodiscovery Scenarios](?wtime=178)
+* [Set Up Annotations](?wtime=210)
+* [Setting Up Multiple Configurations](?wtime=306)
+
+
+Sign up for a trial account here: [https://app.datadoghq.com/signup](https://app.datadoghq.com/signup)
```
Comment (kmshultz) [inline content/videos/tipsandtricks-k8s-autodiscovery.md]: 1) There's already a Free Trial link in the Docs site's header
2) Aside from that header link, we don't usually hawk the free trial in Docs 
3) If we still want to link here, why not have [here] be the link text?

Comment (technovangelist) [inline content/videos/tipsandtricks-k8s-autodiscovery.md]: deleted the phrase. wasn't a fan but its used in other video files. just copied it over

Comment (kmshultz) [inline content/videos/tipsandtricks-k8s-autodiscovery.md]: Gotcha. Should we delete from others for consistency, then?

**Pull request merged**